x/ccv/provider/keeper/throttle.go

```diff
@@ -139,6 +145,8 @@ func (k Keeper) ReplenishSlashMeter(ctx sdktypes.Context) {
 	if meter.GT(allowance) {
 		meter = allowance
 	}
+
+	k.Logger(ctx).Debug("slash meter replenished", "meter", meter.Int64())
```

Comment:
Nit, move to after the setter
```suggestion
	k.Logger(ctx).Debug("slash meter replenished", "new meter value", meter.Int64())
```